internal/credential/vault/repository_credentials.go

```diff
@@ -75,53 +75,55 @@ func (r *Repository) Issue(ctx context.Context, sessionId string, requests []cre
 			return nil, err
 		}
 
-		if cred.getExpiration() < runJobsInterval {
-			event.WriteError(ctx, op,
-				fmt.Errorf("WARNING: credential will expire before job scheduler can run"),
-				event.WithInfo("credential_public_id", cred.GetPublicId()),
-				event.WithInfo("credential_library_public_id", lib.GetPublicId()),
-				event.WithInfo("runJobsInterval", runJobsInterval.String()),
-			)
-		}
+		if !cred.isRevokable() {
```

Comment:
This should be `if cred.isRevokable() {`

Also minor formatting comment, you could remove a bunch of this nested if if you do something like:
```
if !cred.isRevokable() {
  // Amazing comment explaining that we don't need to persist the cred so we can continue early
  creds = append(creds, cred)
  continue
```

Reply:
Yea, I went back and forth between that, which is how I ended up with the incorrect check here :facepalm: